plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -965,13 +965,6 @@ public void createTable(ConnectorSession session, ConnectorTableMetadata tableMe
 
         Table table = buildTable(session, schemaTableName, location, external);
 
-        // Ensure the table has queryId set. This is relied on for exception handling
```

Comment:
This commit seems not related to the current PR

Reply:
It's separate commit. I don't think there is a point in extracting this to a separate pr